tests/integration/test_with_flow.py

```diff
@@ -24,24 +26,18 @@ def test_traversal_path():
         Document(id='chunk112', text=text),
     ]
 
-    f = Flow().add(uses={
-        'jtype': CLIPTextEncoder.__name__,
-        'with': {
-            'default_traversal_paths': ['c'],
-            'model_name': 'ViT-B/32',
-        }
-    })
-    with f:
-        result = f.post(on='/test', inputs=docs, return_results=True)
-        for path, count in [['r', 0], ['c', 3], ['cc', 0]]:
-            assert len(DocumentArray(result[0].data.docs).traverse_flat([path]).get_attributes('embedding')) == count
-
-        result = f.post(on='/test', inputs=docs, parameters={'traversal_paths': ['cc']}, return_results=True)
-        for path, count in [['r', 0], ['c', 0], ['cc', 2]]:
-            assert len(DocumentArray(result[0].data.docs).traverse_flat([path]).get_attributes('embedding')) == count
-
-
-def test_no_documents():
-    with Flow().add(uses=CLIPTextEncoder) as f:
-        result = f.post(on='/test', inputs=[], return_results=True)
-        assert result[0].status.code == 0  # SUCCESS
+    encoder = Executor.load_config('''
```

Comment:
instantiate the class directly, not from `load_config`.

Also move the test to the unittest file

Reply:
done